src/hooks/useHotspots.ts

```diff
@@ -33,6 +149,140 @@ const useHotspots = (): {
     dispatch(fetchCollectables({ account: currentAccount, cluster }))
   }, [cluster, collectables.loading, currentAccount, dispatch, l1Network])
 
+  const fetchingMore = useMemo(() => {
+    if (!currentAccount?.solanaAddress) return false
+
+    return collectables[currentAccount?.solanaAddress].fetchingMore
+  }, [collectables, currentAccount])
+
+  const fetchMore = useCallback(() => {
+    if (
+      !currentAccount?.solanaAddress ||
+      l1Network !== 'solana' ||
+      collectables.loading
+    ) {
+      return
+    }
+    dispatch(
+      fetchMoreCollectables({
+        account: currentAccount,
+        cluster,
+        oldestCollectable: oldestCollectableId,
+      }),
+    )
+  }, [
+    cluster,
+    collectables.loading,
+    currentAccount,
+    dispatch,
+    l1Network,
+    oldestCollectableId,
+  ])
+
+  const createHotspot = useCallback(async () => {
+    if (!currentAccount || !anchorProvider || !currentAccount.solanaAddress)
+      return
+
+    const secureStorage = await getKeypair(currentAccount.address)
+    if (!secureStorage) return
+
+    const owner = new Keypair(secureStorage.keypair)
+    const onboardingKey = random(10)
+    const gateway = await Keypair.makeRandom()
+    const maker = Address.fromB58(
+      '13AjXWhBNWdxq63dSQmPvRx3uQtaa3pMu5wXB1bPUZTTYEiwpgC',
+    )
+
+    await axios.post(
+      'https://onboarding.oracle.test-helium.com/api/v2/hotspots',
+      {
+        onboardingKey,
+        macWlan0: random(10),
+        macEth0: random(10),
+        rpiSerial: random(10),
+        heliumSerial: random(10),
+        batch: 'example-batch',
+      },
+      {
+        headers: {
+          authorization:
+            'pk_cc0LfShOEengfvg/w1y6JLoOQjD090MxDa8N7+rn73w=:sk_IrcEnnhcgI+wSfJfq4w3OZAdF9V++NDQkQSrkpFgXlUp07DbAAdw/QUMbHfJEevQsx0dm1PKwzcUC5Ew6f7YYA==',
+        },
+      },
+    )
+
+    // Sleep for 2 seconds to allow the oracle to create the hotspot
+    await new Promise((resolve) => setTimeout(resolve, 2000))
+
+    const hotspot = (
```

Comment:
Doesnt look like this hotspot var is used? Is it just checking for 404?

Reply:
a bit of copy pasta. Will clean up